docs/api/migration-guides/v2-primitives.mdx

```diff
@@ -0,0 +1,1298 @@
+
+SamplerV2 is simplified to focus on its core task of sampling the output register from execution of quantum circuits. It returns the samples, whose type is defined by the program, without weights. The output data is also separated by the output register names defined by the program. This change enables future support for circuits with classical control flow.
+
+
+## Major changes
+
+### Import
+
+For backward compatibility, you must explicity import the V2 primitives. Specifying `import <primitive>V2 as <primitive>` is not required, but makes it easier to transition code to V2. 
+
+<Admonition type="note">
+After the V1 primitives are deprecated, `import <primitive>` will import the V2 version of the specified primitive.  
+</Admonition>
+
+<Tabs>
+  <TabItem value="EstimatorV2" label="EstimatorV2">
+    ```python 
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+```
+  </TabItem>
+
+  <TabItem value="EstimatorV1" label="Estimator (V1)">
+    ```python 
+from qiskit_ibm_runtime import Estimator
+```
+  </TabItem>
+</Tabs>
+
+<Tabs>
+  <TabItem value="SamplerV2" label="SamplerV2">
+    ```python 
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+```
+  </TabItem>
+
+  <TabItem value="SamplerV1" label="Sampler (V1)">
+    ```python 
+from qiskit_ibm_runtime import Sampler
+```
+  </TabItem>
+</Tabs>
+
+
+### Input and output
+
+#### Input
+
+Both `SamplerV2` and `EstimatorV2` take one or more *primitive unified blocs* (PUBs) as the input. Each PUB is a tuple that contains **one** circuit and the data broadcasted to that circuit, which can be multiple observables and parameters.  Each PUB returns a result.
+
+* SamplerV2 PUB format: (`<circuit>`, `<parameter values>`, `<shots>`), where `<parameter values>`and `<shots>` are optional.
+* EstimatorV2 PUB format: (`<circuit>`, `<observables>`, `<parameter values>`, `<precision>`), where `<parameter values>`and `<precision>` are optional.
+    Numpy [broadcasting rules](https://numpy.org/doc/stable/user/basics.broadcasting.html) are used when combining observables and parameter values.
+    
+Additionally, the following changes have been made:
+
+* EstimatorV2 has gained a `precision` argument in the `run()` method that specifies the targeted precision of the expectation value estimates. 
+* SamplerV2 has the `shots` argument in its `run()` method. 
+* EstimatorV2 does not support resilience level 3.  This is because resilience level 3 in V1 Estimator uses Probabilistic Error Cancellation (PEC), which is proven to give unbiased results at the cost of exponential processing time. You can, however, still use PEC as the error mitigation method by specifying the `pec_mitigation` option.
```

Comment:
This should be moved to the error mitigation section since it's not related to inputs.